modules/transport-netty4/build.gradle

```diff
@@ -42,13 +42,6 @@ dependencies {
     compile "io.netty:netty-transport:4.1.6.Final"
 }
 
-integTest {
-    includePackaged = true
-    cluster {
-        numNodes = 2
```

Comment:
Ah! Fine then. I think we should remove that one in favor of running with two nodes by default but that can be a thing to another PR.